exporters/elasticsearch/src/es_log_exporter.cc

```diff
@@ -100,6 +120,9 @@ class ResponseHandler : public http_client::EventHandler
   std::condition_variable cv_;
   std::mutex mutex_;
 
+  // Whether notify has been called
+  bool stop_waiting_ = false;
```

Comment:
Does this need to be atomic as it could be read/set from multiple threads?

Reply:
I think `stop_waiting_` is already protected by `mutex_`, there's no need to declare `stop_waiting_` as atomic.